CHANGELOG.md

```diff
@@ -141,6 +205,7 @@ Please note that also many other persons usually contribute to the project with
 * `baaa9f42` fix(search): search releases while linking to project
 * `00083ea8` fix(backend): Issues with boolean and timestamp field deserialization and get attachment info REST
 
+
 ## sw360-13.3.0-M1
 
```

Comment:
do we need the description for the **sw360-13.3.0-M1** in the new tagging 15.0.

Reply:
we keep on updating the CHANGELOG.md for new tags/release